### ISSUE TITLE
Data resolution

### ISSUE THREAD

**Issue by sifeng-wu** [JaniceSHLee/chnslab_projectCOVID#1]
Since we will work on different indicators, it is very important that we synchronize our spatial and temporal resolutions to aviod future work. Therefore, I opened this issue so that we have a space to discuss this and to keep each other updated by leaving a comment here.

## Spatial resolution
I guess the best we can do is the national scale based on the availability of #COVID-19 cases. If that's the case, we need to aggregate the environmental indicators to national scale accordingly.

## Temporal resolution
I think we only need to include data after 2020. Before 2020, COVID-19 was only found in China, and the data was not so reliable. I guess daily count of COVID-19 cases are available for most countries. Maybe we try to obtain the daily data for our environmental indicators as well? Seems like the fire data can be obtained at a daily resolution. Not sure whether this is also possible for deforestion data.

**Comment by JaniceSHLee** [comment]
Thanks for beginning this discussion Maple, I agree! 
Spatial resolution - country level
Tempral resolution - Daily may be challenging for deforestation figures. Monthly or bi-weekly is a possibility.

**Comment by sifeng-wu** [comment]
If we are considering montly or bi-weekly data, that means we won't have enough data to do the temporal analysis. For most countries, the COVID-19 cases started to rise after Feb, that means even if we use bi-weekly data, we would have just around 10 data points in each time series at best. Then it seems our analysis will be better off to compare across different countries and montly data will be enough in my opinion. Plus, it will be less of an effort for us to obtain montly data, at least a lot easier for me.  And using monthly data might simplify our following analysis. So, how about we decide on monthly data for now?

**Comment by sifeng-wu** [comment]
About the country boundaries, I recommend using the function *ne_countries* in package *rnaturalearthdata* (see the code below). It contains 241 countries and their geopolygons for the boundaries. And it works with all the *tidyverse* functions very well. The names of the countries could be found under the columns 'name', 'name_long' or 'formal_en'. Maybe we use 'name' for consistency?

```r
library(rnaturalearthdata)
world <- ne_countries(scale = "medium", returnclass = "sf")
```

**Comment by ilubis85** [comment]
> Thanks for beginning this discussion Maple, I agree!
> Spatial resolution - country level
> Tempral resolution - Daily may be challenging for deforestation figures. Monthly or bi-weekly is a possibility.

**Comment by ilubis85** [comment]
> > Thanks for beginning this discussion Maple, I agree!
> > Spatial resolution - country level
> > Tempral resolution - Daily may be challenging for deforestation figures. Monthly or bi-weekly is a possibility.

**Comment by REGE0002** [comment]
Hi all, Lubis and I have been looking, but we are unable to find bi-weekly or monthly deforestation data. We will look for it for 2 -3 days or so (but looks unlikely that we would find something with higher temporal resolution than a year). If not, then we discussed the possibility of using Global Forest Watch data- a dataset that shows deforestation rate per year per country (and even smaller administrative units within countries). We could still use GFW data per country to see if countries with highest deforestation rates (2001-2019) are correlated to high covid-19 cases.

**Comment by JaniceSHLee** [comment]
Thanks Anushka, Lubis! How about GLAD?
![image](https://user-images.githubusercontent.com/29176234/91929645-6d2b4280-ed11-11ea-8336-6bf08a1df86c.png)

**Comment by ilubis85** [comment]
Oke, let me check the GLAD dataset.

**Comment by REGE0002** [comment]
My understanding is that it is only for select countries across tropics? Only Indonesia, Malaysia and Papua New guinea in SEA? We shall check

**Comment by ilubis85** [comment]
Yes the GLAD alerts are operational for land areas between 30 degrees north and south. 
![image](https://user-images.githubusercontent.com/48168202/91931323-bc737200-ed15-11ea-99a4-ee6735cb9dc7.png)

**Comment by ilubis85** [comment]
I was also thinking that it may be difficult to see the correlation between monthly deforestation data with monthly COVID-19 data since deforestation may take years to see have an effect. So maybe the hypothesis to be tested is whether the countries that have high deforestation rates in recent years have high positive case of Covid 19. But it maybe over-shadowed by other variables such as countries GDP.

**Comment by REGE0002** [comment]
Hm.. I was looking at the India portion. One issue with big countries is that they might have high rates of deforestation and high number of cases- but the two might be occuring in separate places. For instance, deforestation rates might be high in western ghats in India but high number of cases might be mostly from the east or Delhi (just an example).

**Comment by REGE0002** [comment]
> My understanding is that it is only for select countries across tropics? Only Indonesia, Malaysia and Papua New guinea in SEA? We shall check

I was referring to this: http://data.globalforestwatch.org/datasets/glad-alerts-footprint
